src/styles/common.scss

```diff
@@ -142,9 +142,8 @@
 /* Hero Banner ****************************************************************/
 
 .heroBanner {
-    --bannerHeight: calc(100vh - var(--headerHeight) - var(--pagePaddingHoriz));
     margin-bottom: var(--pagePaddingHoriz);
-    height: var(--bannerHeight);
+    height: 100vh;
```

Comment:
Something went wrong here, this is already merged, right?

Reply:
merged with development, fixed that